__test__/integrationTests.test.ts

```diff
@@ -279,9 +279,164 @@ describe("Integration tests for AUTH routes:", () => {
             expect(logoutResponse.status).toBe(200);
         });
 
-        test("responds with 401 Unauthorized without session cookie", async () => {
```

Comment:
It seems that this test has ended up at the bottom of the update user tests. It would be ideal if this test remained with the other logout tests. The reason seems to be that the /api/user/update tests describe block is inside the describe block for the logout tests.

Reply:
Is there a duplicate of this test? lines 244-247 and 286-289. This is a logout test so should only be in that section. Maybe we can consider splitting the integration test file at some future date if it starts taking too long to run the tests. test files are run asynchronously but the tests within a file are run synchronously. Not a bit issue at this stage though.

Reply:
I accidentally left 2 extra copies of that test around the place, it now resides in the logout section after the latest commit, thanks for spotting this one! I think splitting the file later on will be good house keeping, especially considering the possibility of adding full browser end to end testing using jest-puppeteer or similar further down the track.

Reply:
Yeah, though e2e tests would likely live on the frontend. Integration tests on the backend are essentially system/e2e tests as they touch on all the separate parts of the web service.